src/modules/dashboard/components/UnclaimedTransfers/UnclaimedTransfersItem.tsx

```diff
@@ -136,7 +126,7 @@ const UnclaimedTransfersItem = ({
           error={ActionTypes.COLONY_CLAIM_TOKEN_ERROR}
           success={ActionTypes.COLONY_CLAIM_TOKEN_SUCCESS}
           transform={transform}
-          disabled={!userHasPermission || !isNetworkAllowed}
+          disabled={!isNetworkAllowed || ethereal}
```

Comment:
So this needs to check if the user has a registered profile as well, not just that it's not an ethereal wallet.

Something like this:
```js
  const { walletAddress, username, ethereal } = useLoggedInUser();

  const hasRegisteredProfile = !!username && !ethereal;
```

This is not necessarily something in the contracts, but it something we don't allow in the dapp, if you don't have your profile registered first.